target_chains/ethereum/entropy_sdk/solidity/scripts/generateAbi.js

```diff
@@ -0,0 +1,54 @@
+const fs = require("fs");
```

Comment:
I copy/pasted this from the other solidity SDK. It's somewhat unfortunate that this code is duplicated, but we don't have a good place for shared javascript and I don't want to solve that problem right now.